Explorer/Assets/DCL/AvatarRendering/Wearables/Systems/ResolveWearableByPointerSystem.cs

```diff
@@ -204,11 +206,28 @@ private void FinalizeWearableDTO([Data] bool defaultWearablesResolved, in Entity
                         catch (AssertionException e) { ReportHub.LogError(new ReportData(GetReportCategory()), $"Cannot apply the DTO to the wearable {component.GetUrn()}: {e.Message}"); }
 
                         component.IsLoading = false;
+                        promise.LoadingIntention.Pointers.Remove(assetEntity.pointers[0]);
```

Comment:
To be honest this flow is hard to follow. We are using `promise.LoadingIntention.Pointers` on a different purpose on which has been created. Perhaps you could declare a new variable, like: `var unresolvedPointers = promise.LoadingIntention.Pointers`.
Also `ReportDTOException` has more responsibilities than how its name is declared, because its not only logging but also resolving the dto with an error. Perhaps rename it to: `ReportAndFinalizeWithError(urn)` ?

Reply:
Changed the name and declared the new pointer array